components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,98 @@
+import { action } from "@storybook/addon-actions";
+import * as React from "react";
+import styled from "styled-components";
+import Card from "./Card";
+import { MOBILE_MEDIA } from "../../constants/mediaquery";
```

Comment:
Please sort imorot statements

Reply:
Components folder is before constants folder, isn't it?